### ISSUE TITLE
Reference fields not recognised 

### ISSUE THREAD

**Issue by MisterY** [authpass/authpass#220]
**Describe the bug**

Keepass reference fields, 
{REF:U@I:4562ce35484370409c7f19eda53a06a5}
are not recognised. When copying the content, the reference gets copied instead of the value of the referenced field. 

**To Reproduce**
1. Create an entry with a username
2. Create another entry with the username field referencing the first entry `{REF:U@I:4562ce35484370409c7f19eda53a06a5}`
3. Go to preview mode
4. Copy the username field by swiping 

**Expected behavior**
The value of the first entry should be copied to the clipboard. 

**Smartphone (please complete the following information):**
 - OS: Android
 - Version 1.7.11

**Comment by hpoul** [comment]
Yes, Reference fields are not yet supported. Shouldn't be too hard to implement though.

https://keepass.info/help/base/fieldrefs.html

**Comment by LuisAPI** [comment]
Yeah. Came here to say I'd love to see it implemented, my password databases are chock full of references.

**Comment by bratkartoffel** [comment]
@hpoul I'm also heavily using the reference fields, using AuthPass is a bit chunky at the moment for me; Any chance to get this feature implemented soon? Thanks!
